src/vivarium/framework/engine.py

```diff
@@ -358,33 +357,39 @@ class Builder:
 
     Attributes
     ----------
-    logging: LoggingInterface
+    configuration : ``LayeredConfigTree``
```

Comment:
I couldn't get this to work so I just kept it nit-picked. Quotes didn't work but for some reason double back-ticks did 🤷‍♂️